translations/hu.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Confirmed": "Fertőzött",
+    "New Cases": "Új esetek",
+    "Deaths": "Elhunytak",
+    "New Deaths": "Mai elhunytak",
+    "Serious" : "Súlyos esetek",
+    "Recovered": "Gyógyult",
+    "Active": "Aktív",
+    "Cases Per M" : "Esetek/millió",
+    "Worldwide" : "Világszerte",
+    "statistic taken at ": "Statisztika utoljára frissítve ",
+    "N/A" : "N/A"
+}
```

Comment:
@radokristof it's all ok, but the country list hungarian translations are missing. 
Please add them in order to have the PR approved.

Reply:
Where are this translations are?